templates/emails/customer-notification-auto-renewal.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * Customer Notification: Notify the customer that an automated renewal their subscription is about to happen.
+ *
+ * @package WooCommerce_Subscriptions/Templates/Emails
+ * @version 1.0.0
+ */
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+/**
+ * @hooked WC_Emails::email_header() Output the email header.
+ *
+ * @since 8.0.0
+ */
+do_action( 'woocommerce_email_header', $email_heading, $email ); ?>
+
+	<p>
+		<?php
+			printf(
+				/* translators: %s: Customer first name */
+				esc_html__( 'Hi %s,', 'woocommerce-subscriptions' ),
+				esc_html( $subscription->get_billing_first_name() )
+			);
+			?>
+	</p>
+
+
+	<p>
+		<?php
+		echo wp_kses(
+			sprintf(
+				// translators: %1$s: name of the blog.
+				_x( 'Your subscription for XYZ on %1$s will be renewed automatically. Thanks for being a loyal customer with us!', 'In customer renewal invoice email', 'woocommerce-subscriptions' ),
```

Comment:
I'm assuming xyz is a placeholder for now but is that going to be the product or something else? 

Just curious because for subscriptions with multiple items, it could get unwieldy.